packages/preview/uo-lth-msc-cs-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "uo-lth-msc-cs-thesis"
+version = "0.1.0"
+compiler = "0.13.0"
+entrypoint = "lib.typ"
```

Comment:
You should move the `lib.typ` file outside of the `template` directory.

Reply:
I want the user to be able to edit it

Reply:
There is no good way to do that at the moment. Depending on what control you want to give them, you can add more parameters to your template function and/or export some parts of your template as variables, allowing them to replace part of it while keeping the rest as it originally was. If your users really want to change everything in the template, you should encourage them to fork your repository and make their own version.